CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 * [CHANGE] Query-frontend: removed `cortex_query_frontend_workers_enqueued_requests_total`. Use `cortex_query_frontend_enqueue_duration_seconds_count` instead. #6121
 * [CHANGE] Ingester: changed the default value for the experimental configuration parameter `-blocks-storage.tsdb.early-head-compaction-min-estimated-series-reduction-percentage` from 10 to 15. #6186
 * [CHANGE] Update Go version to 1.21.2. #6244
+* [CHANGE] Ingester: `/ingester/push/` endpoint has been de-released. #6299
```

Comment:
What does de-released mean? I would use simple words that don't make people think twice:

```suggestion
* [CHANGE] Ingester: `/ingester/push/` HTTP endpoint has been removed. #6299
```